Regex adicionado

Zian25 opened the pull request:
Uma linha de Regex adicionada que remove os caracteres especiais, evitando que o usuário tenha que remover fora da aplicação (Pois a maioria dos CPF's sai padronizado) e facilitando as consultas, realiza com sucesso com e sem caracteres especiais.

Zian25 committed:
Regex adicionado

p0isonBR reviewed (changes requested):
É uma boa adiçao! Só precisa (re)criar a variável, seria uma boa ideia também remover as letras, deixando apenas numeros:
```
cpf = re.sub('[^0-9]+', '', cpf)
```
Faça essas mudanças, que aprovo o pull request. Obrigado!
Edit: Mudanca feita, vou aprovar a pull request, obrigado por contribuir!

p0isonBR committed:
Mudancas na adicao do regex

Craiacao de variavel e mudanca na expressao regular, para aceitar apenas numeros

p0isonBR commented:
Regex que retira a insercao de caracteres invalidos, deixando apenas numeros no input do cpf.

p0isonBR commented:
Pull request aprovada, obrigado por contribuir!
